docs/BasicCommunication/ActivateApp/index.md

```diff
@@ -44,7 +45,7 @@ This request may be sent:
 |:---|:---|:--------|:---------|
 |appID|Integer|true||
 |priority|[Common.AppPriority](../../common/enums/#apppriority)|false||
-|level|[Common.HMILevel](../../common/enums/#hmilevel)|false||
+|level|[Common.HMILevel](../../common/enums/#hmilevel)|false|If the hmi level of this application is to be changed to anything other than `FULL`, then this parameter is included. Should not be `NONE`.|
```

Comment:
I had to read this a few times to understand it. Would it make more sense to say something like "This parameter should only be included if the app is activated to LIMITED?"

Reply:
Ah, this was originally slightly reworded from the HMI API description. I've updated it to be a bit more clear.